src/w3c/linter-rules/required-sections.js

```diff
@@ -40,15 +55,20 @@ export function run(conf) {
     return;
   }
 
+  // We can't check for headers unless we also have a translation
+  if (!localizationStrings[lang]) {
```

Comment:
There's a little caveat here. I think we should normalize the `lang` here like [`getIntlData` does](https://github.com/w3c/respec/blob/0067ad45facb56565463a5fe18b8eb97ffc222a2/src/core/utils.js#L181).

Reply:
yeah, can run it over the same code.